doc/python/webgl-vs-svg.md

```diff
@@ -58,6 +59,15 @@ In addition to the above limitations, the WebGL-powered version of certain SVG-p
 * Range breaks on time-series axes are not yet supported
 * Axis range heuristics may differ
 
+### Multiple WebGL Contexts
+
+*New in 5.19*
+
+If you encounter WebGL context limits when rendering many WebGL-based figures on one page, you can use [Virtual WebGL](https://github.com/greggman/virtual-webgl).
```

Comment:
Can we be more precise, in most contexts I wouldn't think of 5 as "many" but that's typically the limit, right? More than 5 and it'll break without this? We can hedge and say something like "in most browsers it's possible to display up to 5 Plotly WebGL-based figures but any more and you will encounter errors"

Reply:
For me I found in testing that it was 8, but I've seen other mentions of people encountering a limit at 16. And 8-16 is mentioned here https://github.com/greggman/virtual-webgl

Updated the text

Reply:
Oh you could get 8 figures? 16 is the typical number of _contexts_ a browser will allow, but in general each Plotly figure uses more than one context. I thought each one used 3 contexts (16/3 = 5.333 - ie the 6th would not have enough contexts so 5 would be the limit) but I guess either we got that down to 2 at some point, or different plot types (gl3d, gl2d, parcoords, mapbox) use different numbers of contexts.

Reply:
Interesting. Yeah, I had 8 rendering in a notebook in Chrome, and it threw the warning in the console when I ran a 9th cell. I was using this example here: https://plotly.com/python/webgl-vs-svg/#webgl-with-100000-points-with-graph-objects
Does the number of contexts depend on the number of traces?

Reply:
Updated the text with more details on how many figures you would likely be able to render